src/stylesheets/components/_site-search.scss

```diff
@@ -38,21 +38,25 @@ $icon-size: 40px;
   }
 
   .app-site-search__wrapper {
+    $encoded-svg-fill-colour: "%23" + str-slice(ie-hex-str(govuk-colour("dark-grey")), 4);
     display: block;
     position: relative;
     height: $icon-size;
-    background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 36 36' width='40' height='40'%3E%3Cpath d='M25.7 24.8L21.9 21c.7-1 1.1-2.2 1.1-3.5 0-3.6-2.9-6.5-6.5-6.5S10 13.9 10 17.5s2.9 6.5 6.5 6.5c1.6 0 3-.6 4.1-1.5l3.7 3.7 1.4-1.4zM12 17.5c0-2.5 2-4.5 4.5-4.5s4.5 2 4.5 4.5-2 4.5-4.5 4.5-4.5-2-4.5-4.5z' fill='white'%3E%3C/path%3E%3C/svg%3E") govuk-colour("blue") no-repeat top left;
+    background: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 36 36' width='40' height='40'%3E%3Cpath d='M25.7 24.8L21.9 21c.7-1 1.1-2.2 1.1-3.5 0-3.6-2.9-6.5-6.5-6.5S10 13.9 10 17.5s2.9 6.5 6.5 6.5c1.6 0 3-.6 4.1-1.5l3.7 3.7 1.4-1.4zM12 17.5c0-2.5 2-4.5 4.5-4.5s4.5 2 4.5 4.5-2 4.5-4.5 4.5-4.5-2-4.5-4.5z' fill='#{$encoded-svg-fill-colour}'%3E%3C/path%3E%3C/svg%3E") govuk-colour("white") no-repeat top left;
     background-size: $icon-size $icon-size;
   }
 
   .app-site-search__hint,
   .app-site-search__input {
     box-sizing: border-box;
     width: 100%;
-    min-height: $icon-size;
+    height: $icon-size;
     margin-bottom: 0; // BUG: Safari 10 on macOS seems to add an implicit margin.
+    padding: govuk-spacing(1);
+    padding-left: $icon-size -(govuk-spacing(1));
```

Comment:
I think this is meant to be a calculation as `padding-left` only accepts one value, so don't think the brackets around `govuk-spacing(1)` are required?

Reply:
Ah yeah might get away with a space here